include/exadg/utilities/numbers.h

```diff
@@ -24,18 +24,29 @@
 
 namespace ExaDG
 {
-namespace numbers
+namespace types
 {
-unsigned int const steady_timestep = -1;
+using time_step = unsigned int;
 }
 
+namespace numbers
+{
+types::time_step const invalid_timestep = std::numeric_limits<unsigned int>::max();
+types::time_step const steady_timestep  = std::numeric_limits<unsigned int>::max() - 1;
+} // namespace numbers
+
 namespace Utilities
 {
 inline bool
-is_unsteady_timestep(unsigned int const timestep)
+is_unsteady_timestep(types::time_step const timestep)
 {
   return (timestep != numbers::steady_timestep);
 }
+inline bool
+is_valid_timestep(types::time_step const timestep)
+{
+  return (timestep != numbers::invalid_timestep);
+}
```

Comment:
Do we need this function? It looks like it is not used?

Reply:
Same as next comment. I just did this to be consistent but I didnt use it.

Reply:
Now that also trigger_every_timestep is of type types::time_step this function is used